provider/cloudflare.go

```diff
@@ -260,6 +260,23 @@ func newCloudFlareChanges(action string, endpoints []*endpoint.Endpoint, proxied
 }
 
 func newCloudFlareChange(action string, endpoint *endpoint.Endpoint, proxied bool) *cloudFlareChange {
+	if proxied {
```

Comment:
Additionally to my last comment you can also use a simple switch case here:
``
	switch proxied {
	case notSupported[endpoint.RecordType]:
		proxied = false
	}
``